<?xml version="1.0" encoding="UTF-8"?>
<launch>

  <node pkg="marble_frontier_guidance" type="guidance_node.py" name="guidance_controller" output="screen">
    <param name="vehicle_type" type="string" value="ground" />
    <param name="fixed_frame" type="string" value="world" />
    <param name="speed" value="1.0" />
    <param name="Tstar" value="1.0" />
    <param name="reverse" value="false" />

    <!-- <remap from="odometry" to="odometry" /> -->
    <remap from="path" to="ma_goal_path"/>
    <!-- <remap from="path" to="planned_path"/> -->
    <!-- <remap from="goal_pose" to="ma_goal"/> -->
    <remap from="cmd_vel" to="cmd_vel_guidance"/>
    <remap from="lookahead_point" to="lookahead_point"/>
    <remap from="lookahead_vec" to="lookahead_vec"/>
  </node>
</launch>
